cmd/security-proxy-setup/main.go

```diff
@@ -0,0 +1,141 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Tingyu Zeng, Dell
+ * @version: 1.1.0
```

Comment:
This is the version currently reflected in edgex-go
https://github.com/edgexfoundry/edgex-go/blob/4d7ed080f3f3ed8b61b6d20ad0201d97aa64fb6d/version.go#L12

I would advocate removing @version altogether since it will get out of sync right away once DevOps assigns a pipeline-verify job to edgex-go due to semver tagging.

Reply:
Agreed we should just remove it from the license text